### PR TITLE
Step-by-step on adding presence detection with phone

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Oboarding:
- add step-by-step on adding presence detection with phone
- how to set up router-based tracking is out of scope for this PR

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Revamped documentation on zone presence detection to clarify its purpose and enhance user understanding.
  - Introduced a detailed step-by-step guide for setting up zone presence detection using a mobile app.

- **Improvements**
  - Enhanced organization of content with new headings and a comprehensive prerequisites section.
  - Expanded explanations on automation options and methods for implementation, improving user accessibility.

### CODE REVIEW

File: source/getting-started/presence-detection.markdown [hunk 1]
```diff
@@ -1,32 +1,61 @@
...
+### Prerequisites
 
-### Zones
+- [Home Assistant installed](/installation/)
+- [Onboarding steps](/getting-started/onboarding/) completed
+- [Remote access enabled](https://www.home-assistant.io/docs/configuration/remote/)
+  - The easiest way to do this is by enabling
+    - [Home Assistant Cloud](https://nabucasa.com/config/)
+- Mobile phone:
+  - Android phone (Android 5 or later) or iPhone (iOS 10 or later)
```
Comment (bgoncal) [inline source/getting-started/presence-detection.markdown]:
```suggestion
  - Android phone (Android 5 or later) or iPhone (iOS 15 or later)
```

**Review by bgoncal — CHANGES_REQUESTED** [review]

**Review by bgoncal — DISMISSED** [review]

File: source/getting-started/presence-detection.markdown [hunk 2]
```diff
@@ -1,32 +1,61 @@
...
+### Prerequisites
 
-### Zones
+- [Home Assistant installed](/installation/)
+- [Onboarding steps](/getting-started/onboarding/) completed
+- [Remote access enabled](https://www.home-assistant.io/docs/configuration/remote/)
+  - The easiest way to do this is by enabling
+    - [Home Assistant Cloud](https://nabucasa.com/config/)
+- Mobile phone:
+  - Android (Android 5 or later) or iPhone (iOS 10 or later)
```
Comment (bgoncal) [inline source/getting-started/presence-detection.markdown]: Was the change reverted?

Comment (c0ffeeca7) [reply source/getting-started/presence-detection.markdown]: oh! Thank you for the catch!

File: source/getting-started/presence-detection.markdown [hunk 3]
```diff
@@ -1,32 +1,61 @@
...
+   - Select **Continue**.
+3. Go to {% my integrations title="**Settings** > **Devices** & **Services**" %} and look for the new integration that was added: **Mobile App**.
+   - On the integration card, select **1 Device**. This opens the device info page.
+   - You now see your phone name and its entities.
+4. To view the location of your phone on the map, open the **Map** dashboard.
+   - You now see a circle on that map with your initial.
+   - It shows the current location of your phone.
+   - To view the details, select that initial.
+     - Open the **Attributes** list to see the phone's **Latitude**, **Longitude**, and the **Source** of information.
+     - The source is the `device_tracker` entity for that device, for example `device_tracker_pixel_7_pro`.
```
Comment (jpelgrom) [inline source/getting-started/presence-detection.markdown]:
```suggestion
     - The source is the `device_tracker` entity for that device, for example `device_tracker.pixel_7_pro`.
```

Comment (c0ffeeca7) [reply source/getting-started/presence-detection.markdown]: Thank you for the catch!

File: source/getting-started/presence-detection.markdown [hunk 4]
```diff
@@ -1,32 +1,61 @@
...
-It's also possible to run an app on your phone to provide detailed location information to your Home Assistant instance. For iOS and Android, we suggest using the [Home Assistant Companion app][companion].
+## Adding zone presence detection with a mobile phone
 
-During the setup of Home Assistant Companion on your mobile device, the app will ask for permission to allow the device's location to be provided to Home Assistant. Allowing this will create a `device_tracker` entity for that device which can be used in automations and conditions.
+### Prerequisites
 
-### Zones
+- [Home Assistant installed](/installation/)
+- [Onboarding steps](/getting-started/onboarding/) completed
+- [Remote access enabled](https://www.home-assistant.io/docs/configuration/remote/)
```
Comment (frenck) [inline source/getting-started/presence-detection.markdown:26]:
```suggestion
- [Remote access enabled](/docs/configuration/remote/)
```

**Review by frenck — APPROVED** [review]

File: source/getting-started/presence-detection.markdown (hunk 4 above)
Comment (c0ffeeca7) [reply source/getting-started/presence-detection.markdown:26]: > 

#34150
sorry, saw it too late

**Pull request merged**